resources/testharness.js

```diff
@@ -3626,11 +3626,14 @@
 
     Tests.prototype.push = function(test)
     {
+        if (this.phase == this.phases.COMPLETE) {
```

Comment:
We mostly use strict equality in this file, so we should probably remain consistent with that.

This change also isn't immediately obviously part of fixing an off-by-one error.